scripts/generators/patternfly-4-component/templates/component/component.tsx.hbs

```diff
@@ -3,17 +3,19 @@ import styles from '@patternfly/react-styles/css/{{typeDir}}/{{componentName}}';
 import { css } from '@patternfly/react-styles';
 
 interface {{ componentName }}Props {
-  children?: any;
+  /** Content rendered inside {{ componentName }} */
+  children?: React.ReactNode;
+  /** Class to add to outer div */
   className?: string;
 }
 
-export const {{componentName}}: React.SFC<{{componentName}}Props> = ({
+export const {{componentName}}: React.FunctionComponent<{{componentName}}Props> = ({
   className,
   children
-}) => {
+}: {{ componentName }}Props) => {
   return (
-    <div {...props} className={css(styles.{{ camelCase componentName }}, props.className)}>
-      {props.children}
+    <div {...props} className={css(styles.{{ camelCase componentName }}, className)}>
```

Comment:
Can you move the props to be spread at the end.

Reply:
Done!